<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="T3_Add_New_Add_but">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add and Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="T3_Add_New_clear_but">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="T3_Add_New_Sec_le">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Section</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="T3_Add_New_Comment_le">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>671</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Comment</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="T3_Add_New_Brief_le">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Brief</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="T3_Add_New_Error_le">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Error</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="T3_Add_New_Details_le">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Details</string>
   </property>
  </widget>
  <widget class="QComboBox" name="T3_Add_New_Sec_com">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="T3_Add_New_Return_but">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>T3_Add_New_Sec_le</tabstop>
  <tabstop>T3_Add_New_Brief_le</tabstop>
  <tabstop>T3_Add_New_Details_le</tabstop>
  <tabstop>T3_Add_New_Comment_le</tabstop>
  <tabstop>T3_Add_New_Sec_com</tabstop>
  <tabstop>T3_Add_New_Add_but</tabstop>
  <tabstop>T3_Add_New_clear_but</tabstop>
  <tabstop>T3_Add_New_Return_but</tabstop>
  <tabstop>T3_Add_New_Error_le</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
